src/metabase/api/search.clj

```diff
@@ -558,12 +559,30 @@
                              :offset              mw.offset-paging/*offset*
                              :search-native-query search_native_query
                              :verified            verified}))
-        duration   (- (System/currentTimeMillis) start-time)]
-    ;; Only track global searches
-    (when (and (nil? models)
-               (nil? table_db_id)
-               (not archived))
-      (snowplow/track-event! ::snowplow/new-search-query api/*current-user-id* {:runtime-milliseconds duration}))
+        duration   (- (System/currentTimeMillis) start-time)
+        has-advanced-filters (some some?
+                                   [models created_by created_at last_edited_by
+                                    last_edited_at search_native_query verified])]
+    (when (not (nil? context))
+      (cond
+        (and
+         (nil? table_db_id)
+         (not archived))
+        (snowplow/track-event! ::snowplow/new-search-query api/*current-user-id*
+                               {:runtime-milliseconds duration
+                                :context              context}))
+      (cond
+        has-advanced-filters
+        (snowplow/track-event! ::snowplow/search-results-filtered api/*current-user-id*
+                               {:runtime-milliseconds  duration
+                                :content-type          (u/one-or-many models)
+                                :creator               (some? created_by)
+                                :creation_date         (some? created_at)
+                                :last_editor           (some? last_edited_by)
+                                :last-edit-date        (some? last_edited_at)
```

Comment:
I know the casing is converted to snake_case anyway, but this is clojure so lets be consistent and use kebab-case
```suggestion
                                :last-editor           (some? last_edited_by)
                                :last-edit-date        (some? last_edited_at)
```